src/appleseed/renderer/kernel/lighting/subsurfacesampler.h

```diff
@@ -194,18 +194,6 @@ void SubsurfaceSampler::sample(
         if (incoming_point.get_material() == outgoing_material ||
             incoming_point.get_opposite_material() == outgoing_material)
         {
-#ifdef APPLESEED_WITH_OSL
```

Comment:
My idea is running the shader only on the point that was choosen, instead of on all candidate points found by the subsurface sampler (16 in the worst case now), as running the shader could be expensive.
I'm not sure how much of a difference ignoring OSL bump / normal mapping in the subsurface sampler would make. It's something I'd like to check.